### PR TITLE
fuzz testing: add userclk tests

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tests/fuzzing/fuzz_userclk.sh [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+'-v' \
+'-S 0x0' \
+'-S 1' \
+'-B 0x0' \
+'-B 2' \
+'-D 0x0' \
+'-D 3' \
+'-F 0x0' \
+'-F 4' \
+'0000:00:00.0'\
```
Comment (tswhison) [inline tests/fuzzing/fuzz_userclk.sh:67]:
```suggestion
'0000:00:00.0' \
```

File: tests/fuzzing/fuzz_userclk.sh [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+'-H 400' \
+'-L 200' \
+'-H 1600' \
+'-L 1100' \
+'-H 400 -L 200' \
+'-H 1400 -L 200' \
```
Comment (tswhison) [inline tests/fuzzing/fuzz_userclk.sh:73]:
```suggestion
'-H 1400 -L 200'\
```

File: tests/fuzzing/fuzz_userclk.sh [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  local -a long_parms=(\
+'--help' \
+'--version' \
+'--segment 0x0' \
+'--segment 1' \
+'--bus 0x0' \
+'--bus 2' \
+'--device 0x0' \
+'--device 3' \
+'0000:00:00.0'\
```
Comment (tswhison) [inline tests/fuzzing/fuzz_userclk.sh:86]:
```suggestion
'0000:00:00.0' \
```

File: tests/fuzzing/fuzz_userclk.sh [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+'--function 0x0' \
+'--function 4' \
+'--freq-high 400' \
+'--freq-low  200' \
+'--freq-high 1600' \
+'--freq-low  1100' \
+'--freq-high 400 -freq-low  200' \
+'--freq-high 1400 -freq-low  200' \
```
Comment (tswhison) [inline tests/fuzzing/fuzz_userclk.sh:94]:
```suggestion
'--freq-high 1400 -freq-low 200'\
```

File: tests/fuzzing/fuzz_userclk.sh [hunk 5]
```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
```
Comment (tswhison) [inline tests/fuzzing/fuzz_userclk.sh:1]: Please rename the script to be fuzz-userclk.sh, for consistency.

File: tests/fuzzing/fuzz_userclk.sh (hunk 2 above)
Comment (anandaravuri) [reply tests/fuzzing/fuzz_userclk.sh]: fixed

**Review by tswhison — APPROVED** [review]

**Review by tswhison — APPROVED** [review]

**Pull request merged**